src/main/java/thePackmaster/packs/PixiePack.java

```diff
@@ -56,8 +54,10 @@ public static AbstractCard pixieGenerate(Integer cost, Enum color, AbstractCard.
         for (AbstractCard C : CardLibrary.getAllCards())
         {
             if ((cost != null && C.cost != cost) || (color != null && C.color != color) || (type != null && C.type != type)) continue;
-            if (isForeign(C))
-            {
+            if (isForeign(C)
+            && C.rarity == AbstractCard.CardRarity.COMMON || C.rarity == AbstractCard.CardRarity.UNCOMMON || C.rarity == AbstractCard.CardRarity.RARE
```

Comment:
I don't recall Java operator precedence perfectly, but I think you need parentheses around this set of ||s for this logic to work the way you want, since I believe && has higher precedence than ||. So this is probably a bug!

Reply:
Ah, good catch. Fixed.